lib/blanket-options.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+var fs = require('fs-extra');
+
+function exists(path) {
+  return fs.existsSync(path);
+}
+
+function isCurrentFormat(path) {
```

Comment:
I'm assuming modifications to blanket-options will be normal.  As noted above this will only result in a [Yndeh] prompt on upgrade if the contents don't match, correct?